Source/WebCore/storage/StorageNamespaceProvider.h

```diff
@@ -65,6 +71,8 @@ class StorageNamespaceProvider : public RefCounted<StorageNamespaceProvider> {
 
     RefPtr<StorageNamespace> m_localStorageNamespace;
     HashMap<SecurityOriginData, RefPtr<StorageNamespace>> m_transientLocalStorageNamespaces;
+
+    unsigned m_sessionStorageQuota;
```

Comment:
Can we initialize this to 0 instead of using uninitialized memory if setSessionStorageQuota isn't called?

Reply:
Yes, good catch.